recipes/vulkan-loader/all/conanfile.py

```diff
@@ -168,30 +169,24 @@ def package(self):
         cmake = CMake(self)
         cmake.install()
         copy(self, "LICENSE.txt", src=self.source_folder, dst=os.path.join(self.package_folder, "licenses"))
+        rmdir(self, os.path.join(self.package_folder, "lib", "cmake"))
         rmdir(self, os.path.join(self.package_folder, "lib", "pkgconfig"))
         rmdir(self, os.path.join(self.package_folder, "loader"))
 
     def package_info(self):
         self.cpp_info.set_property("cmake_find_mode", "both")
-        self.cpp_info.set_property("cmake_file_name", "Vulkan")
-        self.cpp_info.set_property("cmake_target_name", "Vulkan::Vulkan")
+        self.cpp_info.set_property("cmake_module_file_name", "Vulkan")
+        self.cpp_info.set_property("cmake_file_name", "VulkanLoader")
+        self.cpp_info.set_property("cmake_module_target_name", "Vulkan::Vulkan")
+        self.cpp_info.set_property("cmake_target_name", "Vulkan::Loader")
         self.cpp_info.set_property("pkg_config_name", "vulkan")
         suffix = "-1" if self.settings.os == "Windows" else ""
         self.cpp_info.libs = [f"vulkan{suffix}"]
 
-        # allow to properly set Vulkan_INCLUDE_DIRS in CMake like generators
+        # allow to properly set Vulkan_INCLUDE_DIRS in FindVulkan.cmake
```

Comment:
Vulkan_INCLUDE_DIRS? I think this can be removed since you are no longer using find_package(Vulkan)

Reply:
I'll make an account to comment on the issue.